WORKSPACE

```diff
@@ -24,6 +24,21 @@ load("@google_bazel_common//:workspace_defs.bzl", "google_common_workspace_rules
 
 google_common_workspace_rules()
 
+maven_jar(
```

Comment:
If/when we get around to accepting this, we should import this into bazel-common. But let's hold off on that for nwo